generated/alertsmanagement/log.out

```diff
@@ -83,242 +83,260 @@ warning | PreNamer/DeduplicateName | Deduplicating schema name: 'ErrorResponse'
 warning | PreNamer/DeduplicateName | Deduplicating schema name: 'ErrorResponse' -> 'ErrorResponseAutoGenerated2'
 warning | PreNamer/DeduplicateName | Deduplicating schema name: 'ErrorResponseBody' -> 'ErrorResponseBodyAutoGenerated'
 warning | PreNamer/DeduplicateName | Deduplicating schema name: 'ErrorResponseBody' -> 'ErrorResponseBodyAutoGenerated2'
-info    | autorest.bicep took 1ms
+warning | Skipping path '/subscriptions/{subscriptionid}/providers/microsoft.alertsmanagement/alerts/{alertid}/history': Found mismatch between type segments (2) and name segments (1)
```

Comment:
I believe these warnings are being raised for singleton child resources. I don't believe the `resourceId` function would work on them, so the compiler would need to generate a `concat` expression to generate the full ID before it would make sense to include read-only singleton child resources that follow this naming pattern.